COMPILING.md

```diff
@@ -562,3 +562,29 @@ then run make as follows.
 cd unit
 make test
 ```
+
+## Rust API
+
+CBMC is now offering a [Rust API](src/libcprover-rust/). To build that along with
+CBMC, you need two things:
+
+- Rust/Cargo, instructions the installation of which can be found [here](https://www.rust-lang.org/tools/install), and
+- CMake (the Rust API doesn't support being built with `Make` yet)
```

Comment:
Shouldn't this also specify a version for CMake (unless 3.19 is available on all reasonable systems today)?

Reply:
I will change the documentation and issue a `CMake` error in this case.